deploy/role_binding.yaml

```diff
@@ -1,3 +1,18 @@
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: service-telemetry-operator
+subjects:
+- kind: ServiceAccount
+  name: service-telemetry-operator
+  namespace: service-telemetry
```

Comment:
Good catch! IIRC, since ClusteRoleBindings are not namespaced themselves, you do have to specify the namespace of the subject. I'm not sure how to add templating here. This must be a solved problem, since it's required for any operator that needs ClusterRole perms. I'll hunt around in some other operators to see if I can figure out how they do this.

Reply:
I looked at the openstack-k8s-operator and notice they don't specify a `namespace` with the subject. I wonder if OLM has some fancy built in magic to deal with this at instantiation? https://github.com/openstack-k8s-operators/osp-director-operator/blob/master/config/rbac/cluster_role_binding.yaml

Reply:
I'd probably just try removing it, run the `operator-sdk generate bundle` command to update the CSV, and give it the ol' college try?

Reply:
We've talked this over and will just leave the line as-is - it doesn't seem to cause hard to OLM and is the right value for our testing tools. If we do automated testing in a different namespace this will need adjustment.